<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QLabStatusTableWgt</class>
 <widget class="QWidget" name="QLabStatusTableWgt">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>669</width>
    <height>385</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_7">
   <item>
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="0" column="1">
      <layout class="QVBoxLayout" name="verticalLayout_5" stretch="0,0">
       <property name="rightMargin">
        <number>50</number>
       </property>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <widget class="QLabel" name="label_9">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>50</horstretch>
             <verstretch>20</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>X = 0     </string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="labelZero">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>120</horstretch>
             <verstretch>20</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>0 1 2 3 4 5 6</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <item>
          <widget class="QLabel" name="label_11">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>50</horstretch>
             <verstretch>20</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>X = 1     </string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="labelOne">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>120</horstretch>
             <verstretch>20</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>7 6 5 4 3 2 1</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item row="0" column="2">
      <layout class="QGridLayout" name="gridLayout">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <item row="0" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>Максимальное количество состояний:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEditStatus">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>150</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="maxLength">
            <number>3</number>
           </property>
           <property name="frame">
            <bool>true</bool>
           </property>
           <property name="cursorPosition">
            <number>0</number>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="1" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>Требуемое количество триггеров ( log 2 Nmax )</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEditTrigger">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>150</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="maxLength">
            <number>3</number>
           </property>
           <property name="frame">
            <bool>true</bool>
           </property>
           <property name="cursorPosition">
            <number>0</number>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="0" column="1" rowspan="2">
        <widget class="QPushButton" name="pushButtonCheck_3">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
           <horstretch>150</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Проверить</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QStatusTableWgt" name="tableWidgetStatusTable">
     <property name="font">
      <font>
       <family>Times New Roman</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(0, 0, 0);
gridline-color: rgb(0, 0, 0);
selection-color: rgb(0, 0, 0);
selection-background-color: rgb(0, 255, 0);</string>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <property name="editTriggers">
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
     <property name="alternatingRowColors">
      <bool>true</bool>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::SingleSelection</enum>
     </property>
     <attribute name="horizontalHeaderVisible">
      <bool>false</bool>
     </attribute>
     <attribute name="verticalHeaderVisible">
      <bool>false</bool>
     </attribute>
     <attribute name="verticalHeaderHighlightSections">
      <bool>false</bool>
     </attribute>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Для переключения состояний ячеек используйте: &quot;ЛКМ&quot; (дважды), &quot;Пробел&quot; или &quot;Ввод&quot;</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="pushButtonCheck">
     <property name="text">
      <string>&amp;Проверить</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QStatusTableWgt</class>
   <extends>QTableWidget</extends>
   <header>qlabstatustablewgt.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
